packages/nc-gui/nuxt.config.ts

```diff
@@ -29,7 +29,6 @@ export default defineNuxtConfig({
           name: 'layout',
           mode: 'out-in',
         },
-
```

Comment:
The following meta is moved under app property in the latest version of Nuxt

```
 meta: {
    link: [
      {
        rel: 'icon',
        type: 'image/x-icon',
        href: './favicon.ico',
      },
    ],
    meta: [
      { charset: 'utf-8' },
      {
        name: 'viewport',
        content: 'width=device-width, initial-scale=1',
      },
      {
        hid: 'description',
        name: 'description',
        content: process.env.npm_package_description || '',
      },
    ],
  },
```

Reply:
moving to `app.head`.